src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -96,22 +96,39 @@ class BaseInvertedFlatList extends Component {
     measureItemLayout(nativeEvent, index) {
         const computedHeight = nativeEvent.layout.height;
 
+        // Unclear why but some items will read 0 for height when onLayout runs
+        if (computedHeight === 0) {
+            return;
+        }
```

Comment:
This one stumped me as I see this:

<img width="880" alt="2022-04-19_16-03-12" src="https://user-images.githubusercontent.com/32969087/164132079-e5127fcb-4d57-4f14-aa73-dac92ac6a374.png">

But when logging out items it looks like 0-7 indexes get their layout calculated and the computed height is 0. 

This fix probably isn't what we want since it's conceivable that an item _could_ have a height of 0. And if we don't record a height then we'd never "finish measuring items"